llvm/test/MC/SystemZ/insn-good-z14.s

```diff
@@ -87,6 +88,7 @@
 	bic	0, 524287(%r1,%r15)
 	bic	0, 524287(%r15,%r1)
 	bic	15, 0
+	bi	0
```

Comment:
We already have tests for plain `bi`, so this seems redundant.

Reply:
I agree, will delete.